### PR TITLE
Apply php-cs-fixer fix --rules nullable_type_declaration_for_default_null_value

### PR DESCRIPTION
**Pull request by nicolas-grekas**
| Q             | A
| ------------- | ---
| Branch?       | 5.4
| Bug fix?      | no
| New feature?  |no
| Deprecations? |no
| Issues        | -
| License       | MIT

This one is important to make on 5.4 because it will likely fix a deprecation in a future PHP version, see https://externals.io/message/122214

(do not squash while merging)

### CODE REVIEW

**Comment by nicolas-grekas** [comment]
/cc @keradus FYI

I think we should also make the null default value explicit for untyped properties. What's the fixer for that? :angel:

File: .git-blame-ignore-revs [hunk 1]
```diff
@@ -0,0 +1,2 @@
+# Apply php-cs-fixer fix --rules nullable_type_declaration_for_default_null_value
+f4118e110a46de3ffb799e7d79bf15128d1646ea
```
Comment (fancyweb) [inline .git-blame-ignore-revs:2]: :open_mouth:

Comment (nicolas-grekas) [reply .git-blame-ignore-revs:2]: See https://docs.github.com/en/repositories/working-with-files/using-files/viewing-a-file#ignore-commits-in-the-blame-view ;)

**Comment by lyrixx** [comment]
I really prefer this way! Thanks 👍🏼

**Comment by OskarStark** [comment]
> I really prefer this way! Thanks 👍🏼

Same here, this also makes diffs less noisy when converting to CPP 👏

**Comment by nicolas-grekas** [comment]
Now merged up to 7.1, help wanted to fix tests :)

**Comment by keradus** [comment]
@nicolas-grekas , if you wonder about such fixer, I don't think it exists, feel encouraged to drop the PR for it

```diff
 <?php
 
 class X {
-    public string $name;
+    public ?string $name;
 }
```

**Comment by nicolas-grekas** [comment]
```diff
-    public string $name;
+    public ?string $name;
```

Be careful this wouldn't be equivalent! So no, I won't want this rule anytime :)

**Pull request merged**